tools/packaging/static-build/td-shim/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2022 Intel
+#
+# SPDX-License-Identifier: Apache-2.0
+
+FROM ubuntu:20.04
+ENV DEBIAN_FRONTEND=noninteractive
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        ca-certificates \
+        clang \
+        curl \
+        gcc \
+        git \
+        llvm \
+        nasm && \
+    apt-get clean && rm -rf /var/lib/lists/ && \
+    curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain nightly-2022-04-07 && \
```

Comment:
I read on td-shim's readme why they are using the nightly build of rust. That's ok. However I noticed that since we are pinning on commit "5f62a0e367b1845a54e534d103ed4a697a599ac3", wouldn't be possible to pin the rust version too?

My worry is about the build getting broke on our face from time to time. Apart from that, I suggest to put that toolchain version on `versions.yml` as well, under the td-shim section.

Reply:
Done!